gamemodes/terrortown/gamemode/shared/sh_player_ext.lua

```diff
@@ -367,12 +385,12 @@ function plymeta:GetSubRoleData()
 		return rlsList[i]
 	end
 
-	return INNOCENT
+	return NONE
```

Comment:
Shouldn't this be roles.NONE ?

Also, are the global accessors like `INNOCENT` now removed? This could break many addons, if it is.

Reply:
It has been removed, but rarely used by others, as it was now known nor documented. But yea, as already discussed, this could lead to some incompatibilities.

Reply:
If you say that the removal is fine I'm ok with it. But why aren't you keeping an alias?

Reply:
Most of all unused globals aren't necessary and filling the _G table with trash :D Just because of that, to keep it clean ^^